opentreemap/importer/views.py

```diff
@@ -117,23 +140,26 @@ def _get_table_context(instance, table_name, page_number):
     species = SpeciesImportEvent.objects\
         .filter(instance=instance)\
         .order_by('-created')
-    finished = GenericImportEvent.FINISHED_CREATING
+    inactive_q = Q(is_lost=True) | Q(
```

Comment:
I think it makes sense to leave them in the 'finished' table until the resumption feature is implemented.